guides/source/getting_started.md

```diff
@@ -2363,10 +2363,17 @@ A subscriber may want to unsubscribe at some point, so let's build that next.
 First, we need a route for unsubscribing that will be the URL we include in
 emails.
 
-```ruby
+```ruby#6
+Rails.application.routes.draw do
+  # ...
+  resources :products do
+    resources :subscribers, only: [ :create ]
+  end
   resource :unsubscribe, only: [ :show ]
 ```
 
+The unsubscribe route is added at the top level and is singular in order to handle routes like `/unsubscribe?token=xyz`.
```

Comment:
```suggestion
The unsubscribe route is added at the top level and uses the singular `resource` in order to handle routes like `/unsubscribe?token=xyz`.
```

I'm wondering if `is singular` is enough to emphasize the difference between `resource` and `resources`, what do you think?

Reply:
Much clearer 👍